### PR TITLE
Reenable ppc64le in the devel branch

### PR DESCRIPTION
**Pull request by seisman**
Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by seisman** [comment]
@conda-forge-admin, please rerender

**Comment by seisman** [comment]
@conda-forge-admin, please rerender

**Comment by seisman** [comment]
@conda-forge-admin, please rerender

**Comment by seisman** [comment]
@conda-forge-admin, please rerender

File: recipe/meta.yaml [hunk 1]
```diff
@@ -5,8 +5,8 @@ source:
   git_url: https://github.com/GenericMappingTools/gmt.git
   git_rev: 7f257dc504e455e1a6a6d65e5146cf335ba80b38
 build:
-  number: 0
-  skip: true  # [ppc64le]
+  number: 1
+  skip: true
```
Comment (weiji14) [inline recipe/meta.yaml:9]: Delete this entire line (otherwise it skips everything) and then rerender.

```suggestion
```

**Comment by seisman** [comment]
@conda-forge-admin, please rerender

**Comment by seisman** [comment]
@conda-forge-admin, please restart ci

**Pull request merged**